cms/djangoapps/contentstore/course_group_config.py

```
@@ -6,7 +6,7 @@
 
 from django.utils.translation import ugettext as _
 
-from contentstore.utils import reverse_usage_url
+from .utils import reverse_usage_url
```

Why do we change to the dot import? This is for the elasticsearch import bug? If so, shouldn't we document this inline so that some future developer doesn't go "WHY are we doing relative imports? Let me fix that..."

---

I went for relative imports to fix the tests I wrote for the bug to avoid the silenced exception `No module named contentstore.course_group_config`.

There won't be a future developer for an unmerged PR :sweat_smile:. Additionally "fixing" the relative imports will break the tests.